### ISSUE TITLE
Upgrade cloudflare/circl and google.golang.org/protobuf to reduce vulnerabilities 

### ISSUE THREAD

**Issue by kalpanathanneeru21** [hashicorp/terraform#34897]
### Terraform Version

```shell
1.7.4
```

### Terraform Configuration Files

N/A

### Debug Output

N/A

### Expected Behavior

N/A

### Actual Behavior

```
{
"target": "bin/terraform",
 "type": "gobinary",
"vulnerabilities": [
{
"vulnerability_id": "GHSA-9763-4f94-gfch",
"severity": "HIGH",
"pkg_name": "github.com/cloudflare/circl",
"pkg_path": "",
"installed_version": "v1.3.3",
"fixed_version": "1.3.7",
"cvss_v2_score": "",
"cvss_v3_score": "",
"status_summary": {
"priority": "INFO",
"status": "WARNING"
}
},
{
"vulnerability_id": "CVE-2024-24786",
"severity": "MEDIUM",
"pkg_name": "google.golang.org/protobuf",
"pkg_path": "",
"installed_version": "v1.31.0",
"fixed_version": "1.33.0",
"cvss_v2_score": "",
 "cvss_v3_score": "5.9",
"status_summary": {
"priority": "INFO",
"status": "WARNING"
}
}
],
```

### Steps to Reproduce

build any image with terraform version 1.7.4 and scan with orca will list out about vulnerabilities

**Comment by jbardin** [comment]
Hi @kalpanathanneeru21,

Both of those dependencies are have already been updated, in commits d3b8a55781f and 38119217d91.

Thanks!

**Comment by kalpanathanneeru21** [comment]
In which release of terraform these commits are included

**Comment by jbardin** [comment]
@kalpanathanneeru21, The protobuf change is on `main`, and the `circl` change is included in the v1.8 branch, which will be the next released version. Given that these don't affect the Terraform CLI, there are no additional backports planned.
